### DISCUSSION TITLE
It Is possible to limit the number of the accepted entries?

### DISCUSSION THREAD

**Discussion by Jorman** [Flexget/Flexget#3461] [Q&A] [answered]
Hi, I've a little script that search only for FreeLeech releases, from selected trackers, and put in on download, to help to keep a good ratio.
I already set a limit to the RSS feed on bash script, in this way I try to avoid the FreeLeech day, but now I'm wondering if is possible or there's any way to limit the accepted entries before put on download.

Let's say for example that I've a feed with 100 entries or 25 or whatever, now from these entries not all are accepted, depends on configuration, right? So let's say that 7 of that entries are ready to put on download, but I want to put only the first 3, not more. So in theory Flexget will send to downloading client only 3 entries, not 7 and the very next run is the same and if the entries are the same Flexget will probably put on download the next 3, or if there are some extra entries Flexget will always get the first 3 and so on

I search a bit and I found the Limit plugin and I tried that but this plugin limit the entries before the validation so don't consider the whole RSS file entries.

Any idea about that?

J

**Comment by gazpachoking** [comment, marked as answer]
limit_new plug-in is what you are looking for

**Comment by Jorman** [comment]
Thank you @gazpachoking 
I'll try to get a look to it.
What about if backlog is disabled?
